1-exercises/B-object-arrays/exercise-2.js

```diff
@@ -16,10 +16,13 @@ let store = [
 
 let money = 1.5;
 
-let thingsICanBuy =
-  /* todo */
+let thingsICanBuy = store
+  .filter((itemObject) => {
+    return itemObject.price <= money;
+  })
+  .map((filteredObject) => {
+    return filteredObject.item;
+  });
```

Comment:
You may have noticed that programmers tend to indent their code .Indentation helps us to read our code by visually marking how the code is going to run. And you are really careful of making indent it is really great job keep it up.